### PR TITLE
SEO-2344_Site_Audit_Issues

### PR DESCRIPTION
**Pull request by gowthamanthavasiyappan**
Hi @Aishwarya-Ganesan,

Worked on site audit issues, merge these changes.

Regards,
Gowthaman

### CODE REVIEW

File: js/Electron.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
-title: Getting Started for Electron
-description: How to use Essential JS with Electron.
+title: Getting Started for JavaScript Electron | Syncfusion
+description: Learn here about introduction of Syncfusion Essential JavaScript Electron Control, its elements, and more.
```
Comment (Aishwarya-Ganesan) [inline js/Electron.md:4]: Title mentioned as getting started, description added as introduction.

**Review by Aishwarya-Ganesan — COMMENTED** [review]
Commented

File: js/Electron.md (hunk 1 above)
Comment (gowthamanthavasiyappan) [reply js/Electron.md:4]: Changed

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine,

**Pull request merged**